returns/contrib/pytest/plugin.py

```diff
@@ -28,7 +30,7 @@
 # Also, the object itself cannot be (in) the key because
 # (1) we cannot always assume hashability and
 # (2) we need to track the object identity, not its value
-_ERRORS_HANDLED: Final[Dict[int, Any]] = {}  # noqa: WPS407
+_ErrorsHandled = Dict[int, Any]
```

Comment:
Oh, this is now a type alias. Let's leave a `TODO` item to use explicit `TypeAlias` once `mypy@0.920` is out.

Reply:
I was doubting as well whether to use it. Based on the PEP it seemed optional -- necessary in case of forward references, for example.

On the other hand, if this is looking to be the default way of declaring type aliases (explicit!) then we can already import from `typing_extensions` right?

Reply:
Hmmm something strange in `mypy` indeed, it gives:

```
Variable "typing_extensions.TypeAlias" is not valid as a type
```

mypy doesn't support them yet at all yet?

Reply:
Yes, `0.910` does not support it yet, `0.920` will (hopefully).

Reply:
Is there a other features coming in 0.920 that returns finds useful?

Reply:
I am not sure about `ParamSpec` (it might not be released just yet), but it would remove `decorator` plugin! 🎉